README.md

```diff
@@ -1 +1,43 @@
-# Recreating HackerNews in Angular
\ No newline at end of file
+# Recreating HackerNews in Angular
+
+## Objectives
+
+- Use the HackerNews API to create a clone app
+- Render the top stories view
+- Use ui-router for advanced routing
+- Use dynamic views in routes
+- Set the top stories view as the default route
+- Create relevant Controllers and Services
+- Use relevant Directives to bind ViewModels and Events
+- Use routing resolves to preload data fetched from Services
+- Allow a user to transition to a news item when clicked
+- Show all comments for a post in the news item
+- Nest replies to comments where necessary
+- Display dates in human-readible formats
+- Allow the user to navigate "Next" and "Previous" page
+
+## Instructions
+
+To run the local dev server, you'll need to run `npm install` and then `node server.js`. This will run a local dev server at http://localhost:8080.
+
+HackerNews is a basic news site where people can post items and comment on them. Let's recreate some of it.
+
+Create a state for the "top stories".
+
+We should then grab all the top stories from the URL `https://hacker-news.firebaseio.com/v0/topstories.json`. If you go to that URL, you'll see we actually get lots of IDs - around 500 or so.
+
+Create a service, and a method to grab all the top stories. Then create a method to grab information about a single post by it's ID. This should hit `https://hacker-news.firebaseio.com/v0/item/{id}.json`.
+
+Display a list of 30 posts on the top stories, and put two links at the bottom to allow the user to change to the next/previous page. As we've already got the ~500 or so top stories, we can just slice a different chunk out of the array depending what page they're on instead of making a new request.
```

Comment:
Since we haven't gone too deep into pagination - are there any links to blog posts or resources that we could include for the students?  Or, if the resources are available via Google, tell the students that we expect them to look it up themselves.